src/app/components/plugin-uiframe/plugin-uiframe.component.ts

```diff
@@ -425,6 +425,22 @@ export class PluginUiframeComponent implements OnChanges, OnDestroy {
                 }
                 this.handlePluginInfoRequest(data);
             }
+            if (data.type === "nisq-analyzer-result") {
```

Comment:
The nisq analyzer can and should use the form-submit message! There is no nead for this special message type here.